coderd/agentapi/stats_test.go

```diff
@@ -406,6 +379,136 @@ func TestUpdateStates(t *testing.T) {
 
 		require.True(t, updateAgentMetricsFnCalled)
 	})
+
+	t.Run("WorkspaceUsageExperiment", func(t *testing.T) {
+		t.Parallel()
+
+		var (
+			now = dbtime.Now()
+			dbM = dbmock.NewMockStore(gomock.NewController(t))
+			ps  = pubsub.NewInMemory()
+
+			templateScheduleStore = schedule.MockTemplateScheduleStore{
+				GetFn: func(context.Context, database.Store, uuid.UUID) (schedule.TemplateScheduleOptions, error) {
+					panic("should not be called")
```

Comment:
Use t.Fatal instead for these, otherwise if they do get called they can be harder to track down.